Source/ItemMgr.cpp

```diff
@@ -556,34 +576,77 @@ namespace GW {
         }
 
         bool SalvageSessionCancel() {
-            return SalvageSessionCancel_Func ? SalvageSessionCancel_Func(), true : false;
+            if (!salvage_context)
+                return false;
+            const auto btn = GW::UI::GetChildFrame(GW::UI::GetFrameById(salvage_context->frame_id), {1});
```

Comment:
```suggestion
            const auto btn = GW::UI::GetChildFrame(GW::UI::GetFrameById(salvage_context->frame_id), 1);
```
Why use the function with multiple offsets if you only pass one?